packages/plugins/GeoLocation/src/store/actions.ts

```diff
@@ -132,38 +136,36 @@ const actions: PolarActionTree<GeoLocationState, GeoLocationGetters> = {
       Proj.get('EPSG:4326') as Proj.Projection,
       configuredEpsg
     )
-
-    const boundaryCheckPassed =
-      typeof boundaryLayerId === 'string'
-        ? await passesBoundaryCheck(map, boundaryLayerId, transformedCoords)
-        : containsCoordinate(
-            // NOTE: The fallback is the default value set by @masterportal/masterportalapi
-            configuration?.extent || [510000.0, 5850000.0, 625000.4, 6000000.0],
-            transformedCoords
-          )
-    const boundaryErrorOccurred = typeof boundaryCheckPassed === 'symbol'
-
-    if (
-      boundaryCheckPassed === false ||
-      (boundaryErrorOccurred && boundaryOnError !== 'permissive')
-    ) {
-      dispatch('printPositioningFailed', boundaryErrorOccurred)
-      // if check initially breaks or user leaves boundary, turn off tracking
+    const coordinateInExtent = containsCoordinate(
+      // NOTE: The fallback is the default value set by @masterportal/masterportalApi
+      configuration?.extent || [510000.0, 5850000.0, 625000.4, 6000000.0],
+      transformedCoords
+    )
+    const boundaryCheckPassed = await passesBoundaryCheck(
+      map,
+      boundaryLayerId,
+      transformedCoords
+    )
+    commit('setBoundaryCheckChanged', boundaryCheck !== boundaryCheckPassed)
```

Comment:
The state variable `boundaryCheckChanged` results in undesired behaviour. In the test scenario you defined for Neumünster, when using the Meldemichel setup, we will see two popups. This is due to the use of the state variable being one step behind the setting, since you use this pattern:

```
// ...
    getters: { boundaryCheckChanged } // boundaryCheckChanged is now a local variable with value X
// ...

commit('setBoundaryCheckChanged', boundaryCheck !== boundaryCheckPassed) // the store now holds a potentially new value, but the local boundaryCheckChanged above is still X

// ...

if (boundaryCheckChanged && ... // the old X is used, no matter what the commit sent
```

This becomes visible when using 

```
commit('setBoundaryCheckChanged', boundaryCheck !== boundaryCheckPassed)
console.warn(boundaryCheck !== boundaryCheckPassed, boundaryCheckChanged)
```

and waiting for the second log.

This may be resolved by using `getters.boundaryCheckPassed`, which will look up the newest value. However, I recommend removing `boundaryCheckPassed` from the store altogether and instead modeling it as a local `const`.

Reply:
c55d08dd6b6741809e23d7badc6a5ac368795862

Reply:
@warm-coolguy 🏓 done (I thought the two pop-ups were default behaviour 🙈 )